src/renderer/components/NewProjectModal/ImportMediaView.tsx

```diff
@@ -74,7 +79,33 @@ const ImportMediaView = ({ prevView, closeModal, nextView }: Props) => {
     await addToRecentProjects(project);
 
     nextView();
-  };
+  }, [
+    addToRecentProjects,
+    currentProject,
+    mediaFilePath,
+    nextView,
+    setCurrentProject,
+  ]);
+
+  useEffect(() => {
+    const handleKeypress = async (event: KeyboardEvent) => {
+      if (event.code === 'Enter' && !isAwaitingMedia) {
+        handleTranscribe();
+      }
+    };
+
+    window.addEventListener('keypress', handleKeypress);
+
+    return () => {
+      window.removeEventListener('keypress', handleKeypress);
+    };
+  }, [handleTranscribe, isAwaitingMedia]);
+
+  if (currentProject === null) {
+    return null;
+  }
+
+  const projectName = currentProject.name;
```

Comment:
Not opposed to adding vars to make the tsx more readible. However, might be more readible if the variable was defined at the top of the function, just after `currentProject` is defined. 
Could also use optional chaining (?.) unless we want to fail fast.

Reply:
It cannot be, I cannot have `useDispatch` beneath a conditional statement (really, and early return). For that reason I am considering removing the `projectName` variabel and using `currentProject.name` instead.

```javascript
  const currentProject = useSelector(
    (store: ApplicationStore) => store.currentProject
  );

  const dispatch = useDispatch(); // eslint + react will cry if it is beneath
                                  // the early return since it wont know how many times to render.

  if (currentProject === null) {
    return null;
  }

  const projectName = currentProject.name;
```

Reply:
Oh sorry, I took that code snippet from the wrong branch.

Reply:
Here is the commit for this fix: https://github.com/patrickbrett/mlvet/pull/49/commits/a72a93b92a50ddd3094264aa4feff495f15f4315